apps/clkinfostopw/clkinfo.js

```diff
@@ -21,10 +21,10 @@
         var mins = seconds / 60;
         seconds %= 60;
         if (mins < 60)
-            return "".concat(pad2(mins), "m").concat(pad2(seconds), "s");
+            return "".concat(mins.toFixed(0)+":").concat(pad2(seconds));
         var hours = mins / 60;
         mins %= 60;
-        return "".concat(Math.round(hours), "h").concat(pad2(mins), "m").concat(pad2(seconds), "s");
+        return "".concat(hours.toFixed(0)+":").concat(pad2(mins)).concat(pad2(seconds));
```

Comment:
Here we've removed the separator between minutes and seconds, so they'll be read as a single long number